src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Startup.cs

```diff
@@ -97,5 +72,36 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                     template: "{controller=Home}/{action=Index}/{id?}");
             });
         }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+#pragma warning disable CA1041 // Provide ObsoleteAttribute message
+        [System.Obsolete]
```

Comment:
add a message "Should be fixed during rfactor" to obsolete attribute and remove pragma warning